README.md

```diff
@@ -71,9 +68,9 @@ case class CaseClassA(
   fieldGpb: CaseClassB,
   fieldGpbOption: Option[CaseClassB],
   fieldGpbOptionEmpty: Option[CaseClassB],
-  fieldStringsList: immutable.Seq[String],
-  fieldOptionIntegersList: Option[List[Int]],
-  fieldOptionIntegersEmptyList: Option[List[Int]]
+  fieldStrings: immutable.Seq[String],
+  fieldOptionIntegers: Option[List[Int]],
```

Comment:
Unfortunately repeated fields does not have `has` method...  
This is only an option, you can still have only `List[Int]` and it will just become an empty list.  
Problem with repeated fields in GPB is you have no way how to know whether someone disn't set the field or he had nothing to add there (= put there an empty list).